moveit_setup_assistant/CMakeLists.txt

```diff
@@ -148,7 +121,7 @@ install(TARGETS ${PROJECT_NAME} ${PROJECT_NAME}_widgets ${PROJECT_NAME}_tools ${
   RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
 
 install(DIRECTORY include/ DESTINATION ${CATKIN_GLOBAL_INCLUDE_DESTINATION})
-
+install(DIRECTORY src/ DESTINATION ${CATKIN_GLOBAL_INCLUDE_DESTINATION})
```

Comment:
including the src directory feels wrong...

Reply:
Yes it doesn't feel so right, my reason for that was the I needed to include widgets headers like `setup_screen_widget.h` and `header_widget.h` in the plugins.
What about moving those two headers in a new `dir` like `src/widgets/<new_dir>` and including this `dir` in the `CMakeLists.txt` file instead of just `src`?

Reply:
if you need to share the headers, they belong in the ``include`` folder